src/DeckGlMap/layers/WmsLayer.js

```diff
@@ -0,0 +1,95 @@
+import {CompositeLayer} from '@deck.gl/core';
+import {TileLayer} from '@deck.gl/geo-layers';
+import {BitmapLayer} from '@deck.gl/layers';
+import {load} from '@loaders.gl/core';
+import SphericalMercator from '@mapbox/sphericalmercator';
+
+const conversion = new SphericalMercator({size: 512, antimeridian: true});
+
+class WmsLayer extends CompositeLayer {
```

Comment:
Is there some reason, why use "class" and not arow?

Reply:
Yes, it's recommended https://deck.gl/docs/developer-guide/custom-layers/composite-layers